data/sitesEN.json

```diff
@@ -139,6 +139,16 @@
     "destination_platform": "mediawiki",
     "destination_icon": "hyperrogue.png"
   },
+  
+    "origin": "Lord of the Rings Fandom Wiki",
+    "origin_base_url": "lotr.fandom.com",
+    "origin_content_path": "/wiki/",
+    "destination": "Tolkien Gateway",
+    "destination_base_url": "tolkiengateway.net",
+    "destination_api_path": "/wiki/",
```

Comment:
After a recent change, `destination_api_path` should now be `destination_content_path` (the value is fine)